Mobile/ShopLens/ImageRecognitionMobile/Classificators/WebClassificator.cs

```diff
@@ -20,16 +21,16 @@ public class WebClassificator : IImageClassificator, IAsyncImageClassificator
         /// <inheritdoc/>
         public Dictionary<string, float> ClassifyImage(byte[] image, string cvProjectId, string cvPredictionKey, string cvRequestUri)
```

Comment:
Why are there variables that this method doesn't use, but they're passed to the method anyway?